style/style.scss

```diff
@@ -0,0 +1,294 @@
+    text-decoration: none;
+}
+header, article, #faq, footer {
+    padding:40px;
+}
+header {
+    position: relative;
+    > p {
+    font-size: 1.3em;
+    padding-bottom: 21px;
+    }
+    .teaser_content {
+    color: white;
+    position: absolute;
+    bottom: 232px;
+    left: 133px;
+    }
+    p.teaser_title {
+        font-size: 2.2em;
+        font-family: $font-header;
+    }
+    .teaser_text {
+    width: 65%;
+    padding-bottom: 33px;
+    font-size: 0.93em;
+    }
+    .btn {
+        margin-left:0;
+    }
+}
+nav {
+    ul {
+        margin-top: 26px;
+    }
+    li {
+    float: left;
+    height: 60px;
+    margin-right: 2%;
+    width: 23.5%;
+        &:last-child {
+        margin-right:0;
+        }
+    }
+    span {
+    width: 50%;
+    display: inline-block;
+    }
+    img {
+    width: 19%;
+    padding: 0 18px 0 14px;
+    float: left;
+    }
+    a {  /* I wrestled with the text on nav links to try to get them to vertically center, but didn't spend enough time to figure it out. */
+        &:link, &:visited {
+        background-image: url("../icons/arrow.svg");
+        background-position: 95% center;
+        background-size: 13px auto;
+        background-repeat: no-repeat;
+        background-color: $menu-background;
+        box-sizing: border-box;
+        color: #5A5A5A;
+        font-size: 0.9em;
+        display: block;
+        height: 100%;
+        line-height: 1.3em;
+        vertical-align: middle;
+        padding-top: 10px;
+        }
+        &:hover, &:active, &:focus {
+            background-color: #eee;
+        }
+    }
+    
+}
+img {
+    width: 100%;
+}
+article, footer {
+    background: $bground;
+}
+.max-width {
+    max-width: 905px;
+    margin: 0 auto;
+}
+.left {
+    box-shadow: 2px 2px 4px #DDD;
+    float: left;
+    margin-right: 10%;
+    border: 11px solid white;
+ 
+    -ms-transform: rotate(-3deg); /* IE 9 */
+    -webkit-transform: rotate(-3deg); /* Chrome, Safari, Opera */
+    transform: rotate(-3deg);
+}
+.top {
+    -ms-transform: rotate(3deg); /* IE 9 */
+    -webkit-transform: rotate(3deg); /* Chrome, Safari, Opera */
+    transform: rotate(3deg);
+    margin-right: 0;
+    margin-left: -52%;
+}
+
+.right {
+    box-shadow: 2px 2px 4px #ddd;
+    float: right;
+    margin-left: 10%;
+    border: 3px solid white;
+}
+
+/* There's a small extra space at the bottom of each photo that I wasn't able to fix in the time that I took. */
+.photograph {  
+    width: 40%;
+}
+.thumbBtn {
+    margin-top: 58px;
+    margin-bottom: 98px;
+}
+.thumb {
+    width: 170px;
+    float: left;
+    margin-top: -5px;
+}
+footer {
+    padding-bottom: 100px;
+    ul {
+        padding-top:10px;
+    }
+    li {
+        font-size: 0.8em;
+        line-height: 2.2em;
+    }
+}
+.label {
+    text-transform: uppercase;
+    font-size: 0.8em;
+    margin-bottom: -8px;
+}
+.clearfix {
+    overflow: auto;
+    zoom: 1;
+}
+
+/* Pulled in this code from a button generator website. */
```

Comment:
.btn looks good, but breaks at some sizes 
![image](https://cloud.githubusercontent.com/assets/250996/10886001/5f7f0d0a-813d-11e5-962b-8e1c5da7710c.png)

You don't have to code it up, but thoughts as to how you might prevent this?